TicTacToe.js

```diff
@@ -0,0 +1,320 @@
+        
+      }
+      if(playerNumber == 2){
+        rowTwo[2] = 'O';
+        //testWin(playerNumber);
+        return true;
+      }
+    }
+    else if(rowTwo[2] == 'X' || rowTwo[2] == 'O'){
+      console.log("Spot already taken, please choose a different spot"); 
+      if (playerNumber == 2){
+        playerTwo();
+      }
+      else if (playerNumber == 1){
+        playerOne();
+      }
+    }
+
+  }
+  if(testNumber == 6){
+    if (rowTwo[4] == 6){
+      if(playerNumber == 1){
+        rowTwo[4] ='X';
+        //testWin(playerNumber);
+        return true;
+        
+      }
+      if(playerNumber == 2){
+        rowTwo[4] = 'O';
+        //testWin(playerNumber);
+        return true;
+      }
+    }
+    else if(rowTwo[4] == 'X' || rowTwo[4] == 'O'){
+      console.log("Spot already taken, please choose a different spot"); 
+      if (playerNumber == 2){
+        playerTwo();
+      }
+      else if (playerNumber == 1){
+        playerOne();
+      }
+    }
+
+  }
+  if(testNumber == 7){
+    if (rowThree[0] == 7){
+      if(playerNumber == 1){
+        rowThree[0] =' X';
+        //testWin(playerNumber);
+        return true;
+        
+      }
+      if(playerNumber == 2){
+        rowThree[0] = ' O';
+        //testWin(playerNumber);
+        return true;
+      }
+    }
+    else if(rowThree[0] == 'X' || rowThree[0] == 'O'){
+      console.log("Spot already taken, please choose a different spot"); 
+      if (playerNumber == 2){
+        playerTwo();
+      }
+      else if (playerNumber == 1){
+        playerOne();
+      }
+    }
+
+  }
+  if(testNumber == 8){
+    if (rowThree[2] == 8){
+      if(playerNumber == 1){
+        rowThree[2] =' X';
+        //testWin(playerNumber);
+        return true;
+        
+      }
+      if(playerNumber == 2){
+        rowThree[2] = ' O';
+        //testWin(playerNumber);
+        return true;
+      }
+    }
+    else if(rowThree[2] == 'X' || rowThree[2] == 'O'){
+      console.log("Spot already taken, please choose a different spot"); 
+      if (playerNumber == 2){
+        playerTwo();
+      }
+      else if (playerNumber == 1){
+        playerOne();
+      }
+    }
+
+  }
+  if(testNumber == 9){
+    if (rowThree[4] == 9){
+      if(playerNumber == 1){
+        rowThree[4] =' X';
+        //testWin(playerNumber);
+        return true;
+        
+      }
+      if(playerNumber == 2){
+        rowThree[4] = ' O';
+        //testWin(playerNumber);
+        return true;
+      }
+    }
+    else if(rowThree[4] == 'X' || rowThree[4] == 'O'){
+      console.log("Spot already taken, please choose a different spot"); 
+      if (playerNumber == 2){
+        playerTwo();
+      }
+      else if (playerNumber == 1){
+        playerOne();
+      }
+    }
+
+  }
+  
+  
+}
+function testWin(playerNumber,boxNumber){
+  if (boxNumber == 1 && playerNumber == 1){
+    if(rowOne[0].trim() == 'O' && rowOne[2].trim() == 'O' && rowOne[4].trim() == 'O'){return console.log("Player 1 wins!!!")}
```

Comment:
`return console.log("Player 1 wins!!!")` is the same as 
```
console.log("Player 1 wins!!!");
return;
```
or (because your function is all if-else blocks):
```
console.log("Player 1 wins!!!");
```